### ISSUE TITLE
vxlan router not able to to start in Rhel 7.3 hosts with selinux enabled 

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#9677]
Rancher server version - v1.6.8-rc1 with rancher/net:v0.11.6

Rancher server setup with 3 Rhel 7.3 hosts with selinux enabled and native docker.

vxlan router fails to come up successfully and keeps restarting.

```
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=error msg="Error creating vxlan interface v=&{vtep1042 1042 4789 [14 0 10 42 109 136] 1400}: err=operation not supported"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=error msg="Error creating VTEP interface"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=error msg="couldn't configure: operation not supported"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=error msg="vxlan: Start: failed"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=info msg="vxlan: Reload"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=info msg="vxlan: configure"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=info msg="Listening for ARP requests on eth0"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=info msg="Listening on :8111"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=info msg="vxlan: Reload"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=error msg="Error creating vxlan interface v=&{vtep1042 1042 4789 [14 0 10 42 109 136] 1400}: err=operation not supported"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=error msg="Error creating VTEP interface"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=error msg="vxlan: Reload: couldn't configure: operation not supported"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=error msg="couldn't reload the overlay: operation not supported"
8/14/2017 3:17:39 PMtime="2017-08-14T22:17:39Z" level=fatal msg="operation not supported"
```

**Comment by niusmallnan** [comment]
The vxlan module is not loaded automatically.
```
[ec2-user@ip-172-16-100-190 ~]$ lsmod | grep vxlan
[ec2-user@ip-172-16-100-190 ~]$ sudo modprobe vxlan
[ec2-user@ip-172-16-100-190 ~]$ lsmod | grep vxlan
vxlan                  45114  0
ip6_udp_tunnel         12755  1 vxlan
udp_tunnel             13373  1 vxlan
```
After I manually load the module, everything is normal.
I guess this could be due to Docker1.12.

**Comment by leodotcloud** [comment]
Fixed with image: `rancher/agent:v1.2.6-rc2`.

**Comment by sangeethah** [comment]
Rancher server version - v1.6.8-rc1  with rancher/agent:v1.2.6-rc2.

Rancher server setup with 3 Rhel 7.3 hosts with selinux enabled and native docker.

Vxlan routers are able to start successfully and inter host communication succeeds.
